### PR TITLE
Add helpers which return raw SpiroSegment arrays

### PR DESCRIPTION
**Pull request by terryspitz**
Also add ToString methods on types

### CODE REVIEW

File: drawing.spiro [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (wieslawsoltes) [inline drawing.spiro]: Please remove this file from PR

File: drawing.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (wieslawsoltes) [inline drawing.svg]: Please remove this file from PR

**Comment by wieslawsoltes** [comment]
@terryspitz Can you explain why you need this changes? Also please remove unnecessary files (see my review comments).

**Comment by terryspitz** [comment]
> @terryspitz Can you explain why you need this changes? Also please remove unnecessary files (see my review comments).

This is because I'd like to access the fitted curve, particularly the seg_th at each point, without converting to bezier form.  Please see https://github.com/terryspitz/dactyl-font/blob/master/generate.fs#L271 for my use-case (where line 241 needs the fitted seg_th angle at each original curve point).  Let me know what you think.  (I've removed the files - sorry bad github-fu).

**Review by wieslawsoltes — APPROVED** [review]

**Pull request merged**